ca_description/urdf/create_base_gazebo.xacro

```diff
@@ -3,28 +3,38 @@
   <xacro:macro name="sim_create_base">
 
     <gazebo>
+      <xacro:arg name="enable_roscontrol" default="False"/>
+
       <!-- Diff-drive controller -->
-      <plugin name="differential_drive_controller" filename="libgazebo_ros_diff_drive.so">
-        <alwaysOn>true</alwaysOn>
-        <rosDebugLevel>na</rosDebugLevel>
-        <updateRate>30</updateRate>
-        <leftJoint>wheel_left_joint</leftJoint>
-        <rightJoint>wheel_right_joint</rightJoint>
-        <wheelSeparation>${wheel_separation}</wheelSeparation>
-        <wheelDiameter>${wheel_radius * 2}</wheelDiameter>
-        <wheelTorque>0.5</wheelTorque>
-        <wheelAcceleration>0.5</wheelAcceleration>
-        <commandTopic>cmd_vel</commandTopic>
-        <odometryTopic>odom</odometryTopic>
-        <odometryFrame>odom</odometryFrame>
-        <robotBaseFrame>base_footprint</robotBaseFrame>
-        <publishWheelTF>true</publishWheelTF>
-        <publishOdomTF>true</publishOdomTF>
-        <publishWheelJointState>true</publishWheelJointState>
-        <odometrySource>world</odometrySource>
-        <publishTf>true</publishTf>
-      </plugin>
+      <xacro:if value="${not $(arg enable_roscontrol)}">
+        <plugin name="differential_drive_controller" filename="libgazebo_ros_diff_drive.so">
+          <alwaysOn>true</alwaysOn>
+          <rosDebugLevel>na</rosDebugLevel>
+          <updateRate>30</updateRate>
+          <leftJoint>wheel_left_joint</leftJoint>
+          <rightJoint>wheel_right_joint</rightJoint>
+          <wheelSeparation>${wheel_separation}</wheelSeparation>
+          <wheelDiameter>${wheel_radius * 2}</wheelDiameter>
+          <wheelTorque>0.5</wheelTorque>
+          <wheelAcceleration>0.5</wheelAcceleration>
+          <commandTopic>cmd_vel</commandTopic>
+          <odometryTopic>odom</odometryTopic>
+          <odometryFrame>odom</odometryFrame>
+          <robotBaseFrame>base_footprint</robotBaseFrame>
+          <publishWheelTF>true</publishWheelTF>
+          <publishOdomTF>true</publishOdomTF>
+          <publishWheelJointState>true</publishWheelJointState>
+          <odometrySource>world</odometrySource>
+          <publishTf>true</publishTf>
+        </plugin>
+  </xacro:if>
 
+      <!-- ros-control plugin -->
+      <xacro:if value="$(arg enable_roscontrol)">
+      <plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
+        <robotNamespace>/create1</robotNamespace>
```

Comment:
You have more parameters to parametrize the plugin: https://github.com/ros-simulation/gazebo_ros_pkgs/blob/melodic-devel/gazebo_ros_control/src/gazebo_ros_control_plugin.cpp#L84-L159